lib/utilities/inspec_util.rb

```diff
@@ -99,15 +101,20 @@ def self.parse_data_for_ckl(json)
             data[c_id][:rule_ver]       = control['tags']['stig_id'] unless control['tags']['stig_id'].nil?
             data[c_id][:cci_ref]        = control['tags']['cci'] unless control['tags']['cci'].nil?
             data[c_id][:nist]           = control['tags']['nist'].join(' ') unless control['tags']['nist'].nil?
-            data[c_id][:check_content]  = control['tags']['check'] unless control['tags']['check'].nil?
-            data[c_id][:fix_text]       = control['tags']['fix'] unless control['tags']['fix'].nil?
           end
+
+          unless control['descriptions'].nil?
```

Comment:
Since this is wrapped in a `.nil?` anyway, there may not be a need to use the safe operator with the first find call below. It may be more robust if we say `control['descriptions'].respond_to?(:find)`, in case a string or something that doesn't respond to `find` gets in there.

Also, on the `data[c_id][:cci_ref]`, etc above, it will never be set to nil due to the unless statement on every line. On this code `data[c_id][:check_content]` is set to nil if it isn't found.. Perhaps we need to just remove all these nil checks and just remove all the keys with nil values at the end?